schutzbot/save_journal.sh

```diff
@@ -2,3 +2,8 @@
 
 # use tee, otherwise shellcheck complains
 sudo journalctl --boot | tee journal-log >/dev/null
+
+# As it might contain sensitive information and is important for debugging
+# purposes, encrypt journal-log with a symmetric passphrase.
+gpg --batch --yes --passphrase "$GPG_SYMETRIC_PASSPHRASE" -o journal-log.gpg --symmetric journal-log
```

Comment:
Where does the passphrase come from? I don't see any variable like this in Gitlab CI...

Reply:
Oh yes! I forgot the variable for this project indeed. Adding it now.